### ISSUE TITLE
[FEATURE]: Smarter logs alignment

### ISSUE THREAD

**Issue by eysi09** [garden-io/garden#2478]
### Background / Motivation

Long section names are truncated and hard to ready with the log output. We could probably fix this by aligning the section dynamically based on width. This is what we already do for service logs when running the `garden logs` command.

Something like:

```
✔ short → Getting status 
✔ short → Getting status
✔ somewhat-longer → Getting status
✔ short           → Getting status
✔ very-very-very-long → Getting status
✔ short               → Getting status 
✔ somewhat-longer     → Getting status
```

We could still have some sensible max width and perhaps make that configureable.

**Comment by eysi09** [comment]
Thoughts @edvald?

**Comment by edvald** [comment]
See my comment on #2479. I think it's fine to pad to a certain length and then not trim from there. But sure, a dynamic section width might be a nice improvement if you're feeling like it 🤷

**Comment by eysi09** [comment]
Partially addressed with #2496.
